### PR TITLE
@W-10861572 add data cy

### PR DESCRIPTION
**Pull request by MorPerets**
## Description
Write here a short description

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Are you adding a new package?
- [ ] yes
- [x] no
```
CTC Output:
Case approved, merging is possible.
The case is available at: https://gus.lightning.force.com/lightning/r/Case/500EE00000WMypWYAT/view
Updated at Sun May 01 14:06:16 UTC 2022
```

### CODE REVIEW

File: src/components/TextInput.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ export const TextInput = (props) => {
         </Label>
       )}
       <StyledInput
+        data-cy={'text-input'}
```
Comment (bdimand) [inline src/components/TextInput.js:19]: you can remove the curly brackets form the string and make its data-cy="text-input"

**Review by bdimand — DISMISSED** [review]

File: src/components/TextInput.js (hunk 1 above)
Comment (distroysf) [reply src/components/TextInput.js:19]: also, how will we get this component if there are multiple TextInputs on the page?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@datorama/app-components",
-  "version": "2.11.4",
+  "version": "2.11.5",
```
Comment (distroysf) [inline package.json:3]: run npm install

Comment (MorPerets) [reply package.json:3]: I need to run it on ecom repo no?

Comment (distroysf) [reply package.json:3]: no we need to update the package-lock here as well

File: src/components/TextInput.js [hunk 3]
```diff
@@ -16,6 +16,7 @@ export const TextInput = (props) => {
         </Label>
       )}
       <StyledInput
+        data-cy="text-input"
```
Comment (distroysf) [inline src/components/TextInput.js:19]: also, how will we get this component if there are multiple TextInputs on the page?

File: package.json (hunk 2 above)
Comment (MorPerets) [reply package.json:3]: Yes I will ask someone to do publish for me

File: src/components/TextInput.js (hunk 3 above)
Comment (MorPerets) [reply src/components/TextInput.js]: done

File: src/components/TextInput.js [hunk 4]
```diff
@@ -63,6 +64,7 @@ TextInput.propTypes = {
   type: PropTypes.string,
   maxLength: PropTypes.number,
   autofocus: PropTypes.bool,
+  dataCy: PropTypes.string,
```
Comment (distroysf) [inline src/components/TextInput.js:67]: can you try with hyphen?
found this example.. https://github.com/storybookjs/babel-plugin-react-docgen/pull/23/files#diff-f424801d2039804e394026f71a76edf93f06bda82680b4d145e0a8d508ec0658R19

Comment (MorPerets) [reply src/components/TextInput.js:67]: Yes, it is not working :(

**Review by distroysf — APPROVED** [review]

**Review by edenpessach — APPROVED** [review]

**Pull request merged**